docs/runtimes/configuring-job-resources.md

```diff
@@ -397,9 +413,10 @@ This logic becomes part of the Spark CRD, ensuring that it is consistently appli
 ### Best Practice
 
 Node selection is often used for assigning jobs/pods to GPU nodes. But not all jobs/pods benefit from a GPU node.
-For example, a Databricks “helper” pod runs in a Spark service on Databricks and doesn’t follow the node-selector 
+For example:
+- A Databricks “helper” pod runs in a Spark service on Databricks and doesn’t follow the node-selector 
```

Comment:
is this true @yaelgen / @tomerm-iguazio  ? databricks helper pod is a kubejob so maybe this is meant for the actual databricks job?

Reply:
I think that the "helper pod" is the pod that run in our kubernetes and communicate with databricks (been used as a client only).
we do not run spark on the pod itself.

The job that run in databricks is running on a cluster that can be new or existing one according to user configuration.
There we can run spark:)

About the Node selector, what did you configure on databricks helper pod? @yaelgen

Reply:
looks like there might be a mix up between Spark and Databricks here, that I'm not sure I understand.
For Databricks, the node selector is only relevant for the "helper" pod running in our k8s cluster, as Tomer mentioned. This behaves similarly to how node selectors are applied in Kubejob. It does not affect the actual Databricks cluster, which is not based on Kubernetes and does not run in our cluster, so node selectors have no significance in that context.. maybe this is what this line meant by "doesn’t follow the node-selector"?

Reply:
@yaelgen @tomerm-iguazio @alonmr 
- Updated about Databricks.
- Does it need more explanation about Spark?
- Can you give more examples of where GPU node is "wasted"?